### ISSUE TITLE
Usability improvements

### ISSUE THREAD

**Issue by fulalas** [tsujan/FeatherPad#29]
Shortcuts to close current tab -- this is the default of basically all tab programs, like Opera, Firefox, Notepad++, QtCreator, etc:
1- Ctrl + w;
2- Ctrl + F4;
3- Mouse wheel click;

Shortcuts to open a new tab:
1- Ctrl + t;
2- Mouse wheel click besides last tab;

Also, it would be fine if Esc could close search box.

Ah! Maybe line numbers background could be gray instead of black -- I think this is better for reading.

Congratulations for this program. It's really light and useful! :)

**Comment by tsujan** [comment]
Thanks! I'll think about your suggestions.

Since FeatherPad users may have got used to its shortcuts, it may be better not to change them but to add a shortcut dialog for users to customize (some of) them.

Since I work on Kvantum, FeatherNotes and a few LXQt apps too (especially pcmanfm-qt), it might take a while but I'll tell you when changes are made.

**Comment by fulalas** [comment]
Customization is great. I buy your suggestion. :)

**Comment by tsujan** [comment]
For now, I added closing a tab by middle-clicking it.

I tested adding tabs by clicking after the last tab and found that it was not only unneeded (there's a shortcut and menu/toolbar item for that) but also troublesome (it could interfere with tab DND  if the user inadvertently clicks outside the last tab). I didn't add a "plus" button either -- don't wan't to make the interface heavier an iota for no good reason ;) The code is there but commented out.

Therefore, the main task related to this issue is adding of a custom shortcuts GUI.
